### PR TITLE
fix[WIP]: diagonal dash

### PR DESCRIPTION
**Pull request by MarcGonzalezMoratona**
Diagonal dash has now a similar force than other directions.

We are still facing the issue that the player does not animate when walking after a dash, but it's a previous bug, so don't consider this problem for approving the current PR

### CODE REVIEW

**Comment by MarcGonzalezMoratona** [comment]
Looks good to me. Good job @Piterga12. I can't approve it because I created the PR, so you can approve it in my name.

**Review by Piterga12 — APPROVED** [review]

File: Game/Scripts/PlayerMoveScript.cpp [hunk 1]
```diff
@@ -260,12 +260,30 @@ void PlayerMoveScript::Dash()
...
+	}
 
-	float3 dashImpulse = dashDirection.Normalized() * dashForce;
-	dashImpulse.Normalized();
+	if (dashDirection.z > 0.5f) {
+		dashImpulse.z = dashForce;
+	}
+	else if (dashDirection.z < -0.5f) {
+		dashImpulse.z = -dashForce;
+	}
```
Comment (gerard-queralt) [inline Game/Scripts/PlayerMoveScript.cpp:284]:
```suggestion
	if (dashDirection.z > 0.5f)
	{
		dashImpulse.z = dashForce;
	}
	else if (dashDirection.z < -0.5f)
	{
		dashImpulse.z = -dashForce;
	}
```
😇

**Comment by markmeliaa** [comment]
> We are still facing the issue that the player does not animate when walking after a dash, but it's a previous bug, so don't consider this problem for approving the current PR

This is already solved by #632, so nothing to worry about.

**Review by markmeliaa — APPROVED** [review]
Looks amazing guys, good job!

**Pull request merged**